internal/server/repository_workerauth.go

```diff
@@ -538,7 +535,7 @@ func (r *WorkerAuthRepositoryStorage) findPriorEncryptionKey(ctx context.Context
 	// Create the EncryptionKey using the prior worker auth record
 	priorKey := &types.EncryptionKey{
 		KeyId:           workerAuthSet.Previous.WorkerKeyIdentifier,
-		PrivateKeyPkcs8: workerAuthSet.Previous.ControllerEncryptionPrivKey,
+		PrivateKeyPkcs8: workerAuthSet.Previous.CtControllerEncryptionPrivKey,
```

Comment:
Should this be using the encrypted value?

Reply:
I'm not 100% sure, but from what I could tell, in both structs, the field was generally only used as a ciphertext field. may need to do a little more digging to determine for sure

Reply:
in this case, there is no decryption happening in `FindWorkerAuthByWorkerId`, the function pulling the worker auths, so the value would be encrypted from the db